SecurityPrivacyQuestionnaire.md

```diff
@@ -179,12 +179,57 @@ Yes.
 
 ## [2.17 Do features in your specification enable origins to downgrade default security protections?](https://www.w3.org/TR/security-privacy-questionnaire/#relaxed-sop)
 
+No, there's no ability for specific origins to downgrade default security
```

Comment:
WebRequest provides a way to remove security related headers of embedded content, and CF itself is a workaround of X-Frame-Options and frame-ancestors CSP.

Reply:
Thanks, done.